mobile/lib/widgets/asset_viewer/top_control_app_bar.dart

```diff
@@ -203,26 +205,38 @@ class TopControlAppBar extends HookConsumerWidget {
       actionsIconTheme: const IconThemeData(size: iconSize),
       shape: const Border(),
       actions: [
-        if (asset.isRemote && isOwner) buildFavoriteButton(a),
+        if (asset.isRemote && isOwner && !isReadonlyModeEnabled)

```

Comment:
Instead of hiding the individual component action, maybe just hiding the entire top/bottom control bar would be cleaner.

Reply:
makes sense but I was considering on a what-if scenario if we wanted to implement and show something on read-only mode but I can def. remove it.

Reply:
Oh also the back button is gone if we remove the top control app bar, the back button still works though.